README.md

```diff
@@ -43,15 +43,15 @@ Wouldn’t it be nice if the installer wasn’t so large, so you could actually
 
 ## Usage
 
-You can pass a destination filename as a parameter. Please note that if the download and signature checks succeed, the file will be overwritten without asking. If you don’t supply a filename, a random one in your system’s temp directory will be generated.
+You can pass a destination filename as a parameter. Please note that if the download and signature checks succeed, the file will be overwritten without asking. If you don’t supply a filename, `composer.phar` in the current directory will be used.
 
-Whether you supplied a parameter or not, when Tiny Composer Installer succeeds it will echo the destination filename to standard output and return with an exit code of zero. On error, stdout will be empty and a non-zero error code will be returned. This allows you to do something like this:
+Whether you supplied a parameter or not, when Tiny Composer Installer succeeds it will echo the [`realpath()`](http://php.net/manual/en/function.realpath.php) of the destination filename to standard output and return with an exit code of zero. On error, stdout will be empty and a non-zero error code will be returned. This allows you to do something like this:
 
 ```bash
-phar="$(php tiny-composer-installer.php)" && php "$phar" install && rm "$phar"
+phar="$(php tiny-composer-installer.php "$(mktemp -t)")" && php "$phar" install && rm "$phar"
```

Comment:
On macOS I get the following error (does this work on Linux?):
```bash
mktemp: option requires an argument -- t
usage: mktemp [-d] [-q] [-t prefix] [-u] template ...
       mktemp [-d] [-q] [-u] -t prefix
```

Reply:
Good catch. Haven’t tried it on a Mac, but it does work on Linux. The thing is, there seems to be no way to write one `mktemp` call that works on both Linux and macOS _and_ uses the temp directory. [There are workarounds](https://unix.stackexchange.com/a/84980/10382), but imho they’d only confuse when putting them in the README.

Two options: Leave it as it is (and maybe confuse Mac users) or remove the `mktemp` example altogether. The main reason for it to be there is to demonstrate why it can be useful to output the destination filename, that’s why I think it should stay. I could add something like _“(works on Linux only)”_ though, if you think it’s a good idea?

Reply:
Well, according to [this answer](https://stackoverflow.com/a/31397073), you could provide a template (`mktemp -d "${TMPDIR:-/tmp}/composer.XXXXXXXXX"`, which is more verbose, but works on BSD-derivates as well.

But since this is supposed to be _simple_, I'd agree with just leaving it as it is.